Sources/Views/ListTemplateView.swift

```diff
@@ -286,11 +285,11 @@ class ListTemplateView: UIView {
     private func setupElements() {
         listItems = (0 ... 7).map {
             let item = ListTemplateElementView()
-            item.tag = $0
+            item.tag = Int($0)
```

Comment:
Why we need to convert a number to Int here?

Reply:
unnecessary conversion.removed